src/algorand/AlgorandEngine.ts

```diff
@@ -612,7 +612,7 @@ export class AlgorandEngine extends CurrencyEngine<
       const { txId } = await asyncWaterfall(
         this.networkInfo.algodServers.map(server => async () => {
           const client = new Algodv2('', server, '')
-          return await client.sendRawTransaction(signedTxBytes).do()
+          return client.sendRawTransaction(signedTxBytes).do()
```

Comment:
@typescript-eslint/return-await

Reply:
Weird. keeping the line seems to cause a lint error on my machine. Possibly a vscode bug. I'll revert